app.py

```
@@ -46,7 +46,31 @@ def ping():
+        return Response(str(e), 500)
+    
+@app.route("/comports/active")
+def active_comports():
+    port = ""
+    try:
+        with serial_lock():
+            port = serial_connection.get_port_name()
+
+        if not port:
+            return Response("No connection present.", 204)
+        else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)